packages/field-highlighter/test/user-tags.test.js

```diff
@@ -11,6 +11,7 @@ const user3 = { id: 'c', name: 'baz', colorIndex: 2 };
 async function waitForIntersectionObserver() {
   await nextFrame();
   await nextFrame();
+  await nextFrame();
```

Comment:
Unrelated to this change. Needed to add an extra frame to this helper to make a failing field-highlighter test case pass (currently fails on master also).